### ISSUE TITLE
No enforcement of spaces inside expressions in backticks

### ISSUE THREAD

**Issue by johnbillion** [humanmade/coding-standards#255]
The following is allowed but IMO spaces should be required inside the curly brackets:

```js
`${foo}`
```

Should be:

```js
`${ foo }`
```

**Comment by tfrommen** [comment]
I agree, this should be similar to all the other spaces-required rules. I created #256 to address this.
